src/services/LoginRequest.js

```diff
@@ -0,0 +1,36 @@
+import Cookies from 'js-cookie';
+
+async function LoginRequest(credentials) {
+
+    const getOptions = credentials => {
+
+        const options = {
+          method: 'POST',
+          headers: {
+            'Accept': 'application/json',
+            'Content-Type': 'application/json',
+            'Authorization': null,
+            'X-CSRFToken': Cookies.get('csrftoken')
+          },
+          body: JSON.stringify(credentials),
+        };
+    
+        return options
+      }
```

Comment:
Extract this function outside the block of `loginRequest` and fix the indentation.